dispatcher_plugin_nb2workflow/dataserver_dispatcher.py

```diff
@@ -98,6 +98,52 @@ def test_has_input_products(self, instrument, logger=None):
         query_out.set_done('input products check skipped')
         return query_out, []
 
+    def get_progress_run(self,
+                         call_back_url=None,
+                         run_asynch=None,
+                         logger=None,
+                         task=None,
+                         param_dict=None):
+
+        query_out = QueryOutput()
+        res_trace_dict = None
+
+        if task is None:
+            task = self.task
+        if param_dict is None:
+            param_dict = self.param_dict
+        if run_asynch is not None and run_asynch and call_back_url is not None:
```

Comment:
wouldn't this be the same

```suggestion
        if run_asynch and call_back_url is not None:
```